components/EventBanner.tsx

```diff
@@ -0,0 +1,37 @@
+// CSS
+import styles from "../styles/components/EventBanner.module.scss";
+// Config
+import * as config from "../config.yaml";
+import ReactMarkdown from "react-markdown";
+
+interface Props {
+  color?: string;
+  contextKey?: string;
+  hidden?: boolean;
+}
+
+export default function EventBanner(
+  { color = "event-banner", contextKey = "bannerInfo", hidden = false }: Props,
```

Comment:
ah I think you understood what I was saying wrong here could we have the props passed in be `color, context` and then the consumer of the eventbanner can handle if its hidden or not this would also make it so we dont need to import the config in this module, my thinking is that we might want to use this eventbanner over and over and for different things or even maybe have multiple. 

It might even make sense in a seperate pr if we make a `header` component that bundles the navbar and eventbanner and handles all the logic.

Reply:
I can remove the hidden property.

> It might even make sense in a seperate pr if we make a header component that bundles the navbar and eventbanner and handles all the logic.

lets not be too pedantic about structure of components when setting stuff up. Once the site is up and running I'm more than happy to create those kinds of changes

Reply:
Im 100% fine with that approach, lets keep the hidden prop for now then too.